{"name": "Получение заметки по несуществующему ID", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('http_client', <FixtureRequest for <Function test_get_note_by_nonexistent_id[0]>>)", "trace": "file C:\\Users\\f.palekhov\\Downloads\\NodeApiTEST\\TEST\\tests\\test_api.py, line 174\n      @allure.title(\"Получение заметки по несуществующему ID\")\n      @pytest.mark.parametrize(\"invalid_id\", [-1, 999999, 0])\n      def test_get_note_by_nonexistent_id(self, http_client, base_url, invalid_id):\nE       fixture 'http_client' not found\n>       available fixtures: _attach_last_response, allure_env, base_url, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, doctest_namespace, existing_note_id, http, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Users\\f.palekhov\\Downloads\\NodeApiTEST\\TEST\\tests\\test_api.py:174"}, "parameters": [{"name": "invalid_id", "value": "0"}], "start": 1759764478696, "stop": 1759764478696, "uuid": "8a02af00-2e27-4682-ad57-904c53c9e7eb", "historyId": "3fe5b800079a4cfe409e6cacc1f68719", "testCaseId": "c0de04d895b45b086478706cb4a06724", "fullName": "tests.test_api.TestNoteRetrieval#test_get_note_by_nonexistent_id", "labels": [{"name": "feature", "value": "Управление заметками"}, {"name": "story", "value": "Получение заметок"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestNoteRetrieval"}, {"name": "host", "value": "E4435"}, {"name": "thread", "value": "12124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestNoteRetrieval"]}